src/mqtt/IotaMqttService.cc

```diff
@@ -45,23 +47,80 @@ std::string iota::esp::ngsi::IotaMqttService::publishContextBroker(
   return doPublishCB(apikey, idDevice, jsonMsg);
 }
 
+void iota::esp::ngsi::IotaMqttService::publishMultiAttribute(
+    std::string& multi_payload, std::string& apikey, std::string& idDevice) {
+  std::string sep_medidas = UL20_MEASURE_SEPARATOR;
+
+  std::vector<std::string> tokens_medidas =
+      riot_tokenizer(multi_payload, sep_medidas);
+
+  std::vector<std::string> v_jsons;
+
+  if (tokens_medidas.size() == 0) {
+    // Lanzar excepcion error de protocolo.
+    //
+    std::ostringstream what;
+    what << "Protocol error, no measurements";
+    throw iota::IotaException(iota::types::RESPONSE_MESSAGE_INVALID_PARAMETER,
+                              what.str(),
+                              iota::types::RESPONSE_CODE_BAD_REQUEST);
+  }
```

Comment:
Why use a std::ostringstream here?

It seems this would be enough: 

    throw iota::IotaException(iota::types::RESPONSE_MESSAGE_INVALID_PARAMETER,
                              "Protocol error, no measurements",
                              iota::types::RESPONSE_CODE_BAD_REQUEST);